payments-core/src/main/java/com/stripe/android/paymentsheet/DefaultPrefsRepository.kt

```diff
@@ -10,28 +10,27 @@ import kotlin.coroutines.CoroutineContext
 internal class DefaultPrefsRepository(
     private val context: Context,
     private val customerId: String,
-    private val isGooglePayReady: suspend () -> Boolean,
```

Comment:
usually when we pass values like this we want the value to be lazily read when it's needed as the boolean might be updated after `DefaultPrefsRepository` is created, just want to confirm this won't break any behavior right?

Reply:
Right, but with this change this is not an issue anymore because the value is not used until `getSavedSelection` is called, which is when we already know if it is available or not.